### ISSUE TITLE
Implementation of a projects tag

### ISSUE THREAD

**Issue by tsufz** [MassBank/MassBank-web#88]
We discussed today, that it would be useful, if a record could be related (and searched) for a specific project tag. We are planning to upload a couple of compounds of natural toxins after an embargo from NaToxAq project. Of course, we could add a new DB, but also a project tag would be nice. The idea is the better visibility of our projects and to enhance the willingness of the co-ordinators to support the upload.

**Comment by ChemConnector** [comment]
I would love to make sure that we are getting the chemical set for NaToxAq registered and curated into the CompTox Chemistry Dashboard too! Algal toxins and mycotoxins are of interest to us

**Comment by sneumann** [comment]
I guess what you want is a results page like this:
https://msbi.ipb-halle.de/MassBank/jsp/Result.jsp?type=rcdidx&idxtype=inst&srchkey=LC-ESI-QQ&sortKey=name&sortAction=1&pageNo=1&exec=
but to avoid having to implement a complex project management, this needs to be a simple <KEY>=<VALUE> field in the record. Like: 
```
COMMENT: TAG project=NaToxAq
```
The link would then look like:
https://msbi.ipb-halle.de/MassBank/jsp/Result.jsp?type=rcdidx&idxtype=<KEY>&srchkey=<VALUE>&...
https://msbi.ipb-halle.de/MassBank/jsp/Result.jsp?type=rcdidx&idxtype=project&srchkey=NaToxAq&...

If we change the record format, we can drop the COMMENT: `TAG: project=NaToxAq` (or even leave out the `=`.

**Comment by tsufz** [comment]
@ChemConnector, NaToxAq will be much more than algal and mycotoxins. The goal is to identify as many natural toxins occuring in water as possible, we are talking also about toxins that also originate from terrestrial plants (e.g. artemisa, bracken farm, hogweed etc.). However, your help is welcome!

**Comment by ChemConnector** [comment]
@tsufz It is excellent that the scope of the project encompasses a much broader collection of natural toxins in water. We would definitely be interest. Of late my work has focused on specific cyanotoxins (https://www.epa.gov/ground-water-and-drinking-water/cyanotoxins-drinking-water). In any case I am interested in supporting the efforts with integration to the dashboard as appropriate. Cheers

**Comment by meowcat** [comment]
If we have such a project tag (good idea IMO), we might need a (simple) registry for project names, no?

Also consider a connection to the accession number issue mentioned in #9. I proposed to extend the current scheme (ABC12345 = 3 letter contributor, 5 number ID, or 2+6) to something like 8+8 alphanumeric or even more. The first block (corresponding to the current 2 or 3 letter codes) could then be used for project IDs. So only one registry (where contributors can register a project name and get an unambiguous contributor/project ID for it) would be needed. 

Record could look like this:
```
ACCESSION: ABCD1234EFGH5678
PROJECT: ABCD1234 NaToxAq
```

**Comment by tsufz** [comment]
Actually, I like the ID of a project / contributor registry and 
automated handling of contributor IDs. I think it is a little bit 
outdated to handle that manually.

**Comment by Treutler** [comment]
related to #11

**Comment by tsufz** [comment]
Cool, we need also to implement in RMassBank afterwards. I will create an issue.

**Comment by schymane** [comment]
I find "TAG" is rather uninformative. What about PROJECT_TAG: or just PROJECT:?

**Comment by schymane** [comment]
Btw @tsufz has any progress been made on NaToxAq substances actually becoming available so we can curate and make available on the Dashboard and the NORMAN Suspect Exchange? I have had conversations with many people involved who seem keen in principle but am yet to see anything ...

**Comment by tsufz** [comment]
Yes, I agree with Emma. I would avoid general keyword tags as structured tags are much better for data mining by google and others...

**Comment by tsufz** [comment]
@schymane Nope, NaToxAq compounds are handled as confident so far. We just start to share compounds and will implement an internal MB service soon.

**Comment by tsufz** [comment]
Is there a R script to create the suspect list from basic information, i.e. name, cas, smiles? Or will you do so?

**Comment by tsufz** [comment]
I suggest to maintain the project tags in the new contributors table. I just added two new projects NaToxAq and HBM4EU, I will add a project tag column as well.

**Comment by tsufz** [comment]
https://github.com/MassBank/MassBank-data/blob/master/List_of_Contributors_and_Prefixes.md

**Comment by tsufz** [comment]
@ChemConnector and @schymane , UFZ startet to push first NaToxAq related spectra in a new library called NaToxAq. Nothing special, but a beginning. We will also add compounds related to HBM4EU in future. I already fixed the basic structure (see https://github.com/MassBank/MassBank-data/blob/master/List_of_Contributors_Prefixes_and_Projects.md)

**Comment by tsufz** [comment]
closed with 812c4c7d10ca94425947724ad6f813f875eac5eb
